### PR TITLE
added browserify support

### PR DESCRIPTION
**Pull request by nelsonpecora**

### CODE REVIEW

**Comment by ironspecs** [comment]
This is why I love coveralls.  : )

**Comment by cruzanmo** [comment]
+1

**Comment by ironspecs** [comment]
Not sure about the try-catch.  Looking for another way.

File: index.js [hunk 1]
```diff
@@ -192,8 +192,12 @@ var Module = (function () {
...
         if (this.definitions[name] && typeof this.definitions[name] === 'function') {
           return instantiate.apply(this, [this.definitions[name]].concat(Array.prototype.slice.call(arguments, 1)));
-        } else {
-          throw new Error(name + ' is not defined');
+        } else if (typeof require !== 'undefined') {
+          try {
+            return instantiate.apply(this, [define(this, DSController, name, require(name))].concat(Array.prototype.slice.call(arguments, 1)));
+          } catch (e) {
+            throw new Error(name + ' is not defined');
+          }
```
Comment (ironspecs) [inline index.js]: I think it's just hiding browserify's error.

The Error is a programmer error, for sure, if they're trying to require something that doesn't exist, and the browserify error will be more descriptive than ours, right?  You changed a if/else into a if/elseif.  It should really be a if/elseif/else for when there is no browserify and we still want the error.

**Comment by nelsonpecora** [comment]
So, that line just won't be hit with our tests.

nicholson

**Comment by ironspecs** [comment]
Kay, +1

**Pull request merged**